environments/cooker.json

```diff
@@ -16,5 +16,7 @@
     "business-unit": "Platforms",
     "infrastructure-support": "modernisation-platform@digital.justice.gov.uk",
     "owner": "Modernisation Platform: modernisation-platform@digital.justice.gov.uk"
-  }
+  },
+  "go-live-date": "",
+  "github-oidc-team-name": "github-oidc-repositories"  
```

Comment:
If we're potentially having multiple OIDC values, maybe something like...
```
  "oidc" : {
    "github-team-name": "github-oidc-repositories"
  }
```

Reply:
Ive gone with github-oidc-team-repositories"